src/Codeception/Module/MongoDb.php

```diff
@@ -39,13 +39,20 @@
  * * dsn *required* - MongoDb DSN with the db name specified at the end of the host after slash
  * * user *required* - user to access database
  * * password *required* - password
+ * * dump_type *required* - type of dump
+ *   (one of MongoDb::DUMP_TYPE_JS, MongoDb::DUMP_TYPE_MONGODUMP, MongoDb::DUMP_TYPE_MONGODUMP_TAR_GZ,
```

Comment:
@Naktibalda So how to let the one to use defined constants, not "magic strings"?